INFO     root:get_log.py:36 执行test_login_failed测试用例
INFO     root:get_log.py:36 登录测试-异常数据
INFO     root:get_log.py:36 准备登录
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element {"using": "id", "value": "txtUsername", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element HTTP/1.1" 200 102
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-1/value {"text": "\u8d22\u53cb2sna6763", "value": ["\u8d22", "\u53cb", "2", "s", "n", "a", "6", "7", "6", "3"], "id": "0.6236543734929527-1", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-1/value HTTP/1.1" 200 72
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element {"using": "id", "value": "txtPassword", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element HTTP/1.1" 200 102
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-2/value {"text": "654321", "value": ["6", "5", "4", "3", "2", "1"], "id": "0.6236543734929527-2", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-2/value HTTP/1.1" 200 72
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element {"using": "class name", "value": "sso-btn-login", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element HTTP/1.1" 200 102
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-3/click {"id": "0.6236543734929527-3", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-3/click HTTP/1.1" 200 72
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
INFO     root:get_log.py:36 期望值：True
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element {"using": "xpath", "value": "/html/body/div[2]/div/div/form/div[2]/span", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/element HTTP/1.1" 200 102
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 GET http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-4/displayed {"id": "0.6236543734929527-4", "sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "GET /session/71fa2c05bb1112ceabfff6a431f67322/element/0.6236543734929527-4/displayed HTTP/1.1" 200 72
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request
INFO     root:get_log.py:36 unsupported operand type(s) for -: 'datetime.datetime' and 'builtin_function_or_method'
INFO     root:get_log.py:36 出现异常
INFO     root:get_log.py:36 'WebDriver' object has no attribute 'sava_picture'
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:388 POST http://127.0.0.1:62741/session/71fa2c05bb1112ceabfff6a431f67322/refresh {"sessionId": "71fa2c05bb1112ceabfff6a431f67322"}
DEBUG    urllib3.connectionpool:connectionpool.py:461 http://127.0.0.1:62741 "POST /session/71fa2c05bb1112ceabfff6a431f67322/refresh HTTP/1.1" 200 72
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:440 Finished Request